src/Resources/ResourceManager/Implementation/Policy/GetAzurePolicyAliasCmdlet.cs

```diff
@@ -116,13 +121,23 @@ private bool IsMatch(string input, string match)
         private IEnumerable<Provider> GetAllProviders()
         {
             var returnList = new List<Provider>();
-            var tempResult = this.ResourceManagerSdkClient.ResourceManagementClient.Providers.List(expand: "resourceTypes/aliases");
-            returnList.AddRange(tempResult);
+            var endpoint = new Uri($"{this.ResourceManagerSdkClient.ResourceManagementClient.BaseUri}/providers");
+            var credentials =  AzureSession.Instance.AuthenticationFactory.GetServiceClientCredentials(DefaultContext);
+            var cmdletHeaders = new Dictionary<string, string>
+            {
+                {"ParameterSetName", this.ParameterSetName },
+                {"CommandName", this.CommandRuntime.ToString() }
+            };
+
+            var helper = HttpClientHelperFactory.Instance.CreateHttpClientHelper(credentials, AzureSession.Instance.ClientFactory.UserAgents, cmdletHeaders);
```

Comment:
The right thing to do here is to update the .Net SDK client to return this information,  Please update the .swagger spec / .net SDK client for this extra information,